xclim/indices/generic.py

```diff
@@ -1086,6 +1143,7 @@ def thresholded_statistics(
     Returns
     -------
     xr.DataArray
+        The DataArray for the given thresholded statistic.
```

Comment:
A lot of time "The DataArray [...]" might just be replaced by "DataArray [...]", is there a constraint by numpydoc, or just a choice? If it's your stylistic preference that's all right.

Reply:
It's really minimal, but it's to prevent the styling linter from freaking out that two identical words are following each other ("DataArray DataArray").

Having rewritten many of these now, I find that having full sentence return value descriptions is nicer.

This is entirely a stylistic choice set by me. If you have feelings on this, I can revert.